README.md

```diff
@@ -96,23 +76,26 @@ and debian packages:
 If you are using `Raspbian` you can install any of the `.deb` generated packages.
 If not, just use the compiled binary.
 
+
+### If you have difficulty getting this working with your TFT, please see the wiki page: [TFT Installation Tips](https://github.com/CJeffyB/OctoPrint-TFT/wiki/TFT-Installation-Tips)
```

Comment:
Can you replace `"""` with just a `>`, also the wiki should be at this repository.

Reply:
I'm not seeing the `"""` you noted, so I'm not sure what needs to be replaced, exactly.

As for the wiki, I do not have rights to edit or add anything to your repository, and there is currently no wiki there.  That is why it is on my git.  I could not figure out how to do a pull request for the wiki, and am guessing it will have to be created by you, then copied and pasted...